source/ConfigSetup.h

```diff
@@ -78,7 +78,7 @@ EMP_BUILD_CONFIG(SymConfigBase,
 
     GROUP(SGP, "Complex Genomes Settings"),
     VALUE(CYCLES_PER_UPDATE, size_t, 4, "Number of CPU cycles that organisms run every update"),
-    VALUE(THREAD_COUNT, size_t, 12, "Number of threads used to process organisms in parallel"),
+    VALUE(THREAD_COUNT, size_t, 4, "Number of threads used to process organisms in parallel"),
```

Comment:
What is this change from?

Reply:
My personal computing was getting very loud when I was running it on the twelve so I lowered it to 4 for my own sake